source/change-streams/tests/README.rst

```diff
@@ -43,6 +43,9 @@ Each YAML file has the following keys:
     Valid topologies are ``single``, ``replicaset``, and ``sharded``.
   - ``changeStreamPipeline``: An array of additional aggregation pipeline stages to add to the change stream
   - ``changeStreamOptions``: Additional options to add to the changeStream
+
+    - ``batchSize``: The batch size to apply to the change stream
```

Comment:
Is this needed? Typically, options do not get documented in the spec test format since they map to an Options class/struct defined in the spec. I realize this spec's format differs from transactions/CRUD, but I think this can be consistent with how we work with `options` elsewhere.

Reply:
I added it because I looked at the Java driver test runner and it didn't support any change stream options at all, and figured that would be the case for other test runners as well.  But will remove if you prefer it.  I can see how for CRUD tests we wouldn't want to enumerate them all.

Reply:
Done.